src/Im2Col.h

```diff
@@ -70,7 +70,7 @@ template <>
 void im2col<1>(const int channels,
                const std::vector<net_t>& input,
                std::vector<float>& output) {
-    constexpr unsigned int boardsize = 19;
+    constexpr unsigned int boardsize = BOARD_SIZE;
```

Comment:
Same. Pointless now.

Reply:
I don't understand this, the variable is used and the function itself is called somewhere in the code, so it doesn't seem useless to me, but if I put back 19, it still works (despite the following assert that should stop the program). 

Should I remove the 3 lines ?
```CPP
    constexpr unsigned int boardsize = 19;
    auto outSize = size_t{channels * boardsize * boardsize};
    assert(output.size() == outSize);
```

Reply:
I mean that it's setting boardsize==BOARD_SIZE.

Just use BOARD_SIZE.